emmo/patch.py

```diff
@@ -176,23 +201,79 @@ def namespace_init(self, world_or_ontology, base_iri, name=None):
         self.name = self.name[:-4]
 
 
-# Inject methods into Owlready2 classes
-setattr(ThingClass, '__dir__', _dir)
-setattr(ThingClass, 'get_preferred_label', get_preferred_label)
-setattr(ThingClass, 'get_parents', get_parents)
-setattr(ThingClass, 'get_annotations', get_class_annotations)
-setattr(ThingClass, 'disjoint_with', disjoint_with)
-setattr(ThingClass, 'get_indirect_is_a', get_indirect_is_a)
-
-setattr(PropertyClass, 'get_preferred_label', get_preferred_label)
-setattr(PropertyClass, 'get_parents', get_parents)
-setattr(PropertyClass, 'get_annotations', get_property_annotations)
-
-setattr(Restriction, 'get_typename', get_typename)
-
 setattr(Namespace, '__init__', namespace_init)
 
-# Method names for individuals must be different from method names for classes
-type.__setattr__(Thing, 'get_preflabel', get_preferred_label)
-type.__setattr__(Thing, 'get_individual_annotations',
-                 get_individual_annotations)
+
+#
+# Extending Metadata
+# ==================
+def keys(self):
```

Comment:
Does this take the ontology as argument? emmo.ontology.Ontology does not have .namespace.